docs/src/api/class-websocket.md

```diff
@@ -54,3 +54,33 @@ Event name, same one would pass into `webSocket.on(event)`.
   - `timeout` <[float]> maximum time to wait for in milliseconds. Defaults to `30000` (30 seconds). Pass `0` to disable timeout. The default value can be changed by using the [`method: BrowserContext.setDefaultTimeout`].
 
 Either a predicate that receives an event or an options object. Optional.
+
+## method: WebSocket.waitForFrameReceived
+* langs: csharp, java
+- returns: <[WebSocketFrame]>
```

Comment:
What is WebSocketFrame? I see `[string]|[Buffer]` above.

Reply:
There is no way to defined such type union in java so I use a dedicated type for that. Will be happy to change that to something more elegant, let's discuss offline.